### DISCUSSION TITLE
Custom cards/fields on cards in resource grid?

### DISCUSSION THREAD

**Discussion by cmaxw** [avo-hq/avo#3154] [Help] [answered]
I'm working on building out a system that lists events. In the grid view, I'd like to show the date and time as well as some other information in addition to the title and description/body. 

Do I do that with a partial card? If so, how do I tell the resource to use that card instead of the default?

Or, is there a way to add more fields to the default resource cards displayed?

**Comment by Paul-Bob** [comment, marked as answer]
There is no way right now to specify what partial / card to use. You can eject ([docs here)](https://docs.avohq.io/3.0/eject-views.html#--component) and maintain the whole component until there is a native way of doing it. (`rails generate avo:eject --component Avo::Index::GridItemComponent`)

@adrianthedev made this PR to pass extra fields to the card https://github.com/avo-hq/avo/pull/2650 but is closed for now.

**Reply by adrianthedev** [reply]
But specifying which component to use could be a feature.
Please open an issue so we can track it.

**Reply by Paul-Bob** [reply]
Let's continue the discussion here https://github.com/avo-hq/avo/issues/3155
